packages/localizations/src/en-US.ts

```diff
@@ -24,8 +24,10 @@ export const enUS: LocalizationResource = {
   formFieldLabel__phoneNumber_username: 'phone number or username',
   formFieldLabel__emailAddress_phoneNumber_username: 'Email address, phone number or username',
   formFieldLabel__password: 'Password',
+  formFieldLabel__currentPassword: 'Current Password',
```

Comment:
```suggestion
  formFieldLabel__currentPassword: 'Current password',
```

Reply:
@panteliselef I still see the `Password` capitalized. Do we want to keep this, or you missed it?